universities/university-of-michigan.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "University of Michigan"
+link: "https://open.umich.edu/"
+--- 
+
+# University of Michigan
+
+- *OSPO*: Yes, in the Library.
+- *Personnel*: Ashwathi Iyer
+- *Link*: https://open.umich.edu/find-open-resources-u-m
+- *Member of*: [HELIOS](https://www.heliosopen.org/members)
+
+## General Description
+
+Through Open Michigan, the University of Michigan community can share its research, teaching, and creative output with the outside world. At the University of Michigan, we are the center for everything open, offering services and knowledge related to open publishing, open data, and open educational resources.
+
+## Core Objectives
+
+- Open Michigan encourages scholars and students to maximize the impact of their work through open sharing. As a public institution, the University of Michigan is committed to serving our university community, the residents of the state of Michigan, and the broader world in which we live. 
+
+## Primary Contacts
+
+- Ashwathi Iyer, the Assistant Director of Software Licensing & Research Partnerships
+
+## Other context
+
+After a dormant period, today Open Michigan functions as a collaboration between the University of Michigan’s Health Information Technology and Services (HITS) group and the University of Michigan Library. HITS continues to manage the OER collection and provide support to faculty who wish to share their materials online, while the Library maintains a gateway to all things open at the University of Michigan, including open access journals and open data.
```

Comment:
"Today"? What do you mean by that? 

This seems to be only about the library - is any open source mentioned here?